### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by hoanhan101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+- Configuration Error
+
+Each VLAN is unique to a Virtual Circuit in a sense that when it is assigned to a
+Virtual Circuit, it can not exist in another Virtual Circuit without first being 
+removed.
+
+## Installation
+
+```
+pip install netbox-virtual-circuit-plugin
```
Comment (edaniszewski) [inline README.md:22]: is this project hosted on pypi?

File: README.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+Each Virtual Circuit is identified by a name, a numeric ID (0-32767), along with
+a context. Each must be assigned one of the following operational statuses:
+- Pending Configuration
+- Configured
+- Pending Deletion
+- Configuration Error
+
+Each VLAN is unique to a Virtual Circuit in a sense that when it is assigned to a
+Virtual Circuit, it can not exist in another Virtual Circuit without first being 
+removed.
```
Comment (edaniszewski) [inline README.md:17]: I'm not sure I understand what this sentence is saying

File: README.md (hunk 1 above)
Comment (hoanhan101) [reply README.md:22]: Yes that’s the plan

**Comment by hoanhan101** [comment]
Good point. I will open a separate issue to keep track of this. As for now, my only thought for publishing it to pypi is that it would save us a ton of time to install and enable the plugin in Netbox, though I might not know if there's also another good way to do so.

**Pull request merged**